components/chart/search/textbox.tsx

```diff
@@ -68,25 +75,30 @@ export const Textbox = ({
   autoCompleteList,
   onBlur,
   containerClassName,
+  onListItemClick
 }: BCTextboxProps) => {
   const [showAutoComplete, setShowAutoComplete] = useState(false);
+  const autoCompleteListNames = autoCompleteList.map(el => {
```

Comment:
I think it would be better to pass autoCompleteList as key/value pairs to avoid parsing again the id and name here

Reply:
done